### PR TITLE
break: re-define Media Query Breakpoints and layout width

### PR DESCRIPTION
**Pull request by tujoworker**
Why?

We need to change/align our breakpoints in code. This will help everyone work with confidences that components and page templates cross team will work together.

In code the new breakpoints will be:

- small: 640px / `40em`
- medium: 960px / `60em`
- large: 1152px / `72em`

For now, we skip `x-large` until someone needs it.

BREAKING CHANGE

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.md [hunk 1]
```diff
@@ -30,6 +30,30 @@ v10 of @dnb/eufemia contains _breaking changes_. As a migration process, you can
...
+- **xx-large:** `1280` is now `1440` – and `80em` is now `90em`
+- **x-large:** `1152` is now `1280` – and `72em` is now `80em`
+- **large:** `960` is now `1152` – and `60em` is now `72em`
+- **medium:** `800` is now `960` – and `50em` is now `60em`
+
+1. Find `$layout-x-large` and replace with `$layout-large`
+1. Find `$layout-xx-large` and replace with `$layout-x-large`
+1. Find `--layout-x-large` and replace with `--layout-large`
+1. Find `--layout-xx-large` and replace with `--layout-x-large`
+
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.md:46]:
```suggestion
Find `--layout-large` and replace with `--layout-medium`
```

Should we add this as well?
If this is correct, then we should probably go over our code and replace `--layout-large` with `--layout-medium`.

File: packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md [hunk 2]
```diff
@@ -221,20 +230,17 @@ You can re-use the SASS mixins from Eufemia:
...
+@media screen and (max-width: 60em) {
   /* medium */
 }
-@media screen and (max-width: 60em) {
-  /* large (default) */
+@media screen and (max-width: 72em) {
+  /* large */
 }
-@media screen and (min-width: 60em) and (max-width: 72em) {
+@media screen and (min-width: 80em) {
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md:241]: Would it improve the example by using the var instead?  🤔 
```suggestion
@media screen and (min-width: var(--layout-x-large)) {
```

**Review by langz — APPROVED** [review]
Looks great 🤩

File: packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md:241]: Sadly, var can not be used in CSS – yet.

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.md (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.md:46]: We do not really use them actually, we use `--layout-large` it a [single place](https://github.com/dnbexperience/eufemia/blob/3c8ba8b1b70acadadaee70fdccbd5131691abce6/packages/dnb-eufemia/src/components/global-error/style/dnb-global-error.scss#L25).

File: packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md]: I have update this doc file, to not include x-large 👍

**Pull request merged**